### DISCUSSION TITLE
Model Training Problems

### DISCUSSION THREAD

**Discussion by Enverrrr** [open-mmlab/mmocr#1689] [General]
Hello I have a problem about model training. I wanna train recognition model I read concerned documentations but I am confused. Does anyone help me about this?

**Comment by gaotongxiao** [comment]
Could you elaborate on the part that confuses you? You can also try following [quick run](https://mmocr.readthedocs.io/en/dev-1.x/get_started/quick_run.html) first to understand how everything typically works.

**Reply by Enverrrr** [reply]
I followed quick run and I've done but when I tried SAR recognition model training I've taken data/icdar2013/textrecog_train.json' . I'm sorry I'm confused. What can I do now?

**Reply by gaotongxiao** [reply]
Did you mean the file `data/icdar2013/textrecog_train.json` was missing? You can run the following command:
```bash
python tools/dataset_converters/prepare_dataset.py icdar2013 --task textrecog
```
to prepare the dataset. Read more at https://mmocr.readthedocs.io/en/dev-1.x/user_guides/data_prepare/dataset_preparer.html

**Reply by Enverrrr** [reply]
I've done however when I'm trying to prepare icdar2011 I'm getting 'UserWarning: icdar2011 is not supported yet.' this one. Can I do something else?

**Reply by Enverrrr** [reply]
I deleted all train and test list except icdar 2013 and 2015 in config file but It's not working

**Reply by gaotongxiao** [reply]
icdar2011 is not supported by preparer yet, so currently you'll have to follow the steps [here](https://mmocr.readthedocs.io/en/dev-1.x/user_guides/data_prepare/det.html#icdar-2011-born-digital-images) and prepare it manually.

For the config part, you can find some explanation at https://mmocr.readthedocs.io/en/dev-1.x/user_guides/config.html#config. Hope it helps!
